### ISSUE TITLE
How to start tmux so it restores automatically

### ISSUE THREAD

**Issue by iovis9** [tmux-plugins/tmux-continuum#52]
I just installed the plugin and it doesn't restore when I run `tmux` in my terminal.

### Relevant config ([Full file](https://github.com/iovis9/dotfiles/blob/master/.tmux.conf))
tmux 2.8
```sh
## Plugins
set -g @plugin 'tmux-plugins/tmux-continuum'
set -g @plugin 'tmux-plugins/tmux-resurrect'
set -g @plugin 'tmux-plugins/tpm'

## Plugin config
set -g @continuum-restore 'on'
set -g @continuum-save-interval '10'
set -g @resurrect-capture-pane-contents 'on'
set -g @resurrect-strategy-nvim 'session'

run -b '~/.tmux/plugins/tpm/tpm'
```

### Findings
Digging through the code I noticed my issue was in the following line:
https://github.com/tmux-plugins/tmux-continuum/blob/master/continuum.tmux#L54

When I run `tmux` without a server running it starts one session called `0`. When changing that line to expect one session running (`[ "$(number_of_sessions)" -eq 1 ]`) it ran the restore without issues.

### Question
Am I running the wrong command for starting tmux?

**Comment by badjware** [comment]
I have the same issue than you.

While your workaround makes tmux-continuum retore on server start it also makes it restore on config reload if you have only a single session. This is undesirable as it effectively rollback your session a few minute everytime you reload your config.

I poked around a bit and found a solution. Instead of checking the number of session, it would be better to check how old the server is. If the age of the server is over a certain threshold, say 10 seconds, allow the restore to happen.

Here is a new condition to do this: `[ "$(tmux ls -F '#{start_time}')" -gt "$(($(date +%s)-10))" ]`

I am running the plugin with the patched condition and it it works flawlessly.

**Comment by iovis9** [comment]
Neat! Thanks man, that looks great. I'm going to throw it into my local installation for now

**Comment by kenny1983** [comment]
I'm having this issue too, but neither @iovis9's or @badjware's suggestion is working for me. Also, by looking into the continuum.tmux and resurrect.tmux scripts I was able to determine the scripts that *they* use to implement their functionality. But running those directly doesn't work either!!

Perhaps it has something to do with the fact that I'm actually running Linux through Windows' WSL...that could add a whole host of complexities to the issue!

Nonetheless, I'd love to hear more suggestions on how to fix this! MTIA :-)

**Comment by Hologos** [comment]
@bruno- Is gonna gonna get fixed? Thank you

**Comment by bruno-** [comment]
Hi all,
sorry for the wait, this should now be fixed via 4e2279688d8a577ed14f42c4e8af1acb58100a57
Thanks @badjware, I used your solution and mentioned you in the commit.

**Comment by bruno-** [comment]
I'll close this issue now, but please open another one if you still experience problems.
